_posts/2019-08-14-matt-mitchell.md

```diff
@@ -0,0 +1,20 @@
+---
+author: Rebecca Kindschi
+category: General
+date: 2019-08-14 06:00:00
+layout: post
+image: /static/img/social/presenters/matt-mitchell.png
+title: "Announcing Our Keynotes: Matt Mitchell"
+---
+
+We're excited to announce our final; keynote speaker for DjangoCon US 2019: Matt Mitchell!
```

Comment:
Is this colon supposed to be here?

Reply:
Nope, I'll remove. Good catch, thank you!